scripts-test/weather.py

```
@@ -0,0 +1,60 @@
+    response.raise_for_status()  # This will raise an HTTPError if the request failed
+    weather_data = response.json()
+    return weather_data
+
+def print_weather(weather_data):
+    print(f"Weather in {weather_data['nearest_area'][0]['areaName'][0]['value']}:")
+    print(f"Current condition: {weather_data['current_condition'][0]['weatherDesc'][0]['value']}")
+    print(f"Temperature: {weather_data['current_condition'][0]['temp_C']}°C / {weather_data['current_condition'][0]['temp_F']}°F")
+    print(f"Feels like: {weather_data['current_condition'][0]['FeelsLikeC']}°C / {weather_data['current_condition'][0]['FeelsLikeF']}°F")
+    print(f"Humidity: {weather_data['current_condition'][0]['humidity']}%")
+
+def main():
```

Ruff (0.8.2)

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)